### PR TITLE
Adjust D2k encyclopedia descriptions for light and heavily armored units

### LINKED ISSUE #20649: Sonic Tank armor type Encyclopedia inconsistency
I have noticed that both sonic tank and devastator units have this line in encyclopedia:

```
They are very resistant to bullets and high explosives, but vulnerable to missiles and high caliber guns.
```

But they have different armor types:

Sonic Tank:
```
	Armor:
		Type: light
```

Devastator
```
	Armor:
		Type: heavy
```

Don't know what is the correct way to resolve the inconsistency.

### PR DESCRIPTION
**Pull request by abcdefg30**
Suggestions to resolve #20649.

### CODE REVIEW

File: mods/d2k/rules/vehicles.yaml [hunk 1]
```diff
@@ -649,7 +649,7 @@ combat_tank_o:
 	Turreted:
 		TurnSpeed: 20
 	Encyclopedia:
-		Description: The Combat Tank is effective against most vehicles, less so against lightly armored vehicles.\n\nThe Ordos Combat Tank is faster but weaker.
+		Description: The Combat Tank is effective against most vehicles, less so against lightly armored vehicles.\n\nThe Ordos Combat Tank is faster than the Atreides Combat Tank, but weaker.
```
Comment (MustaphaTR) [inline mods/d2k/rules/vehicles.yaml]: This reads weird to me, Harkonnen one is slower than Atreides one, so naturally Ordos one is the fastest. But this sounds like it is only faster than Atreides one. `than other Combat Tanks`, or `than its counterparts` would better imo. And maybe even use superlatives `The Ordos Combat Tank is the fastest variant of the Combat Tank, but it is also the weakest.`

**Comment by penev92** [comment]
This will be the first time we release the encyclopedia, so we should try to make it as accurate as possible before the release. Adding the label.

**Comment by abcdefg30** [comment]
Updated.

**Review by pchote — APPROVED** [review]
ok

**Comment by penev92** [comment]
Why does the first commit change the descriptions to 
1. "light explosives"
2. "explosives"
3. "explosives"
4. "small explosives"

if they are all supposed to be the same?
Or are they not supposed to be the same?

**Comment by abcdefg30** [comment]
2. "explosives, to a lesser degree"
3. "and to some degree, explosives"

So they boil down to the same. Some context from Discord:
image
image
image
image

**Review by penev92 — APPROVED** [review]
OK

**Comment by penev92** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/3f86cc0b908728f09c2d6ae35806a0c4f5af9ec9...e7e508d639c95b6d1b9f5d9f43782608d4e0e852)

Commits on `prep`: 88134c073f038784bd3b9cdd392d6852d2992d71, a8fc826fc9bab94339e171db0c6fe30b0a13d805

**Pull request merged**